src/graphql/graphql.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common'
+import { GraphqlService } from './graphql.service'
+import { GraphQLClient } from 'graphql-request'
+
+@Module({
+  controllers: [],
+  providers: [GraphqlService],
+  imports: [GraphQLClient],
```

Comment:
I don't think you need this line, `imports` are for when you're trying to utilize providers from other modules in nestJS: https://stackoverflow.com/a/73751722/282250